src/vivarium_conic_lsff/components/observers.py

```diff
@@ -132,4 +132,98 @@ def get_transition_count(pop, config, disease, transition, event_time, age_bins)
                                                         year=event_time.year)
     base_filter = QueryString('')
     transition_count = get_group_counts(transitioned_pop, base_filter, base_key, config, age_bins)
-    return transition_count
\ No newline at end of file
+    return transition_count
+
+
+
+class LiveBirthWithNTDObserver:
```

Comment:
For specialized observers, I'll typically drop the abstraction layer in configuration defaults.  So have 
```
configuration_defaults = {
    'metrics: {
        'live_births_with_ntds': {
            'by_year': True,
            'by_sex': True,
        }
    }
}
```

And then skip the whole __init__ method.